cli/azd/pkg/containerapps/container_app.go

```diff
@@ -30,13 +45,15 @@ type ContainerAppService interface {
 		subscriptionId,
 		resourceGroup,
 		appName string,
+		options *ContainerAppOptions,
 	) (*ContainerAppIngressConfiguration, error)
```

Comment:
Do we need to update the return type to be something "weakly typed" so we don't drop things when options points to a newer API version?